lms/static/js/learner_dashboard/views/program_card_view.js

```diff
@@ -0,0 +1,32 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'text!../../../templates/learner_dashboard/program_card.underscore'
+           ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             programCardTpl
+         ) {
+            return Backbone.View.extend({
+                className: 'program-card',
+                tpl: _.template(programCardTpl),
+                initialize: function() {
+                    this.render();
+                },
+                render: function() {
+                    if (this.model.has('marketingUrl')){
```

Comment:
This is a bad condition. I will change it.
However, I want to test the empty model case. See the last test case here: https://github.com/edx/edx-platform/blob/schen/ECOM-3194/lms/static/js/spec/learner_dashboard/program_card_view_spec.js#L34
How do I make my code satisfy that last test case?